config/manager/manager.yaml

```diff
@@ -28,6 +28,11 @@ spec:
       affinity:
         nodeAffinity:
           preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 2
+              preference:
+                matchExpressions:
+                  - key: node-role.kubernetes.io/infra
+                    operator: Exists
```

Comment:
Does it make sense to add `value: reserved` in the toleration/taint as it was suggested in [OCP](https://docs.openshift.com/container-platform/4.13/machine_management/creating-infrastructure-machinesets.html#:~:text=add%20tolerations%20for%20the%20pod%20configurations%20you%20want%20to%20schedule%20on%20the%20infra%20node%2C%20like%20router%2C%20registry%2C%20and%20monitoring%20workloads.%20add%20the%20following%20code%20to%20the%20pod%20object%20specification%3A)?

Reply:
IIRC taints are [unique by key and effect](https://github.com/medik8s/fence-agents-remediation/pull/53#discussion_r1230839930), but maybe using the value is desired? I am unfamiliar with infra nodes and their usage thus I am unaware of the other values that might be in use.

Reply:
I don't see any value in adding the value (pun intended 😉 )